bin/kafka-streams-application-reset.sh

```diff
@@ -18,4 +18,4 @@ if [ "x$KAFKA_HEAP_OPTS" = "x" ]; then
     export KAFKA_HEAP_OPTS="-Xmx512M"
 fi
 
-exec $(dirname $0)/kafka-run-class.sh org.apache.kafka.tools.StreamsResetter "$@"
+exec $(dirname $0)/kafka-run-class.sh kafka.tools.StreamsResetter "$@"
```

Comment:
Ok, so it sounds like the conclusion is leave it as is now, we agree that nothing about the tools other than the shell script is public API, and eventually we'll move it to one of the other packages.

We should definitely think about making tools like these have accessible and reliable Java APIs as folks end up relying on them anyway since they can't always shell out to do these things and may want it integrated into their app (e.g. so they only need to pass a command line flag to do the reset before starting the app).